django-verdant/docker/run-dev.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+django-admin compress --force
```

Comment:
Shouldn't be required for dev

Reply:
Going to ignore for now as I can't get the `docker-compose` set up working locally.